tests/test_policy.py

```diff
@@ -14,3 +16,5 @@ def test_policy_checks_glibc():
     assert policy > POLICY_PRIORITY_LOWEST
     policy = versioned_symbols_policy({"some_library.so": {"GLIBC_999"}})
     assert policy == POLICY_PRIORITY_LOWEST
+    policy = versioned_symbols_policy({"some_library.so": {"OPENSSL_1_1_0"}})
+    assert policy == POLICY_PRIORITY_HIGHEST
```

Comment:
Would you mind adding one more test asserting `SYMBOLWITHOUTUNDERSCORES` satisfies `POLICY_PRIORITY_HIGHEST` or something similar?